src/Persist.jl

```diff
@@ -798,4 +798,17 @@ function readmgr(jobname::AbstractString)
     mgr::JobManager
 end
 
+"Fetch job result"
+function fetch(jobname::AbstractString)
+    resultfile = joinpath(jobdirname(jobname), resultfilename(jobname))
```

Comment:
There's a `readmgr` function to read a manager back in. You could define
```Julia
fetch(jobname::AbstractString) = fetch(readmgr(jobname))
```
and maybe similarly for `getstdout`, `getstderr`, `cleanup` etc.